partner_contact_in_several_companies/models/multi_contact.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, _, api
+from openerp.osv import expression
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    contact_type = fields.Selection([('standalone', _('Standalone Contact')),
```

Comment:
Move the first argument line to the second line, and thus you only need to indent the line one level, being less constrained for putting all the text:
```
contact_type = fields.Selection(
    [('standalone', _('Standalone Contact')),
    ...
 ```